tests/rptest/tests/many_partitions_test.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2020 Vectorized, Inc.
```

Comment:
2022

Reply:
Updated year.  The company name I think we're leaving the same until someone does a bulk replace with whatever the official form of the new name is.